openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTGCSConfig.tsx

```diff
@@ -358,7 +337,7 @@ export const DBTGCSConfig: FunctionComponent<Props> = ({
         <label
           className="tw-block tw-form-label tw-mb-1"
           htmlFor="gcs-cred-path">
-          {requiredField('GCS Credentials Path')}
+          GCS Credentials Path
```

Comment:
We make all these field as non-required?

Reply:
@chirag-madlani yes, we do as per new schema!